### PR TITLE
Cleanup repo

### PR DESCRIPTION
**Pull request by manno**
Three small things:

* remove .vscode
* add golangci-linter (additionally)
* switch version.go to LDFLAGS

https://github.com/epinio/epinio/projects/1#card-58835502

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package version
+
+var Version = "v0.0.0-dev"
```
Comment (andreas-kupries) [inline version/version.go:3]: TODO: Remove this file from `.gitignore`.

Comment (manno) [reply version/version.go:3]: Just checked and I believe it's removed: https://github.com/epinio/epinio/pull/274/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L18

File: .golangci.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+run:
```
Comment (jimmykarily) [inline .golangci.yml:1]: How do we run the linter locally? Shouldn't it run as part of `make lint`?

File: version/version.go (hunk 1 above)
Comment (andreas-kupries) [reply version/version.go:3]: :facepalm: Apologies. I seem to be very out of it this morning. Hopefully not for the entire day. I should know to look at the entire PR changes, not just one commit. Retracting.

File: .golangci.yml (hunk 2 above)
Comment (manno) [reply .golangci.yml:1]: `golangci-lint run` - https://golangci-lint.run/usage/install/#local-installation

I didn't want to change `make lint`, because that seems opinionated and is used by CI to fail the PR.

There are some discussions about linting and if it should fail CI. You'll always have to exclude edge cases, especially when you go beyond linking into static analysis. Which is very useful for coding, but you might not want to fail tests just because cyclomatic complexity checks don't understand ginkgo contexts...
On the other hand, I'm going to fix the linter errors (spellcheck, errchck) anyhow, why not fail the run then.

Didn't want to decide that by myself. Maybe we merge this and look at the results from golangci-lint on Github and decide which ones should fail PRs together?

**Review by jimmykarily — APPROVED** [review]

**Comment by jimmykarily** [comment]
While CI is busted, locally tests pass:

```
[1] 
[1] Ran 29 of 29 Specs in 2159.608 seconds
[1] SUCCESS! -- 29 Passed | 0 Failed | 0 Flaked | 0 Pending | 0 Skipped
[1] PASS
[2] Deleting tmpdir on node 2
[2] 
[2] 
[2] Summarizing 1 Failure:
[2] 
[2] [Fail] Epinio enable/disable features services-incluster [It] enables minibroker services 
[2] /home/opensuse/workspace/go/pkg/mod/github.com/onsi/gomega@v1.10.5/internal/asyncassertion/async_assertion.go:56
[2] 
[2] Ran 13 of 13 Specs in 2198.963 seconds
[2] SUCCESS! -- 13 Passed | 0 Failed | 1 Flaked | 0 Pending | 0 Skipped
[2] PASS
```

Merging.

**Review by jimmykarily — APPROVED** [review]
